### ISSUE TITLE
What messages can carry MAX_CACHE_DURATION?

### ISSUE THREAD

**Issue by martinduke** [moq-wg/moq-transport#553]
The other subscribe parameters are really specific about what messages can carry them. This one isn't.

**Comment by ianswett** [comment]
I think it's only valid in SUBSCRIBE_OK given that's the only messages that corresponds to cacheable Objects being delivered?

**Comment by fluffy** [comment]
Yes, agree with your logic. I think this could use a PR to make this clear.
